chia/util/db_version.py

```diff
@@ -0,0 +1,14 @@
+import aiosqlite
+
+
+async def lookup_db_version(db: aiosqlite.Connection) -> int:
+    try:
+        cursor = await db.execute("SELECT * from database_version")
```

Comment:
In the past, I've created a "global value" table to keep these one-off values (basically a key/value store). I guess it depends on how magic you want to make the `database_version` table.

Reply:
I'm not familiar with some special support for storing a single value like this, other than in a single-row table. I did some research, but couldn't come up with anything. Do you have a reference?

Reply:
Nothing fancy, see https://github.com/richardkiss/pycoin/blob/main/pycoin/wallet/SQLite3Persistence.py#L84

It's just a key/value pair table. But it's useful for global state type things, like database version or "last block hash validated".